site/en/docs/webstore/program_policies/index.md

```diff
@@ -23,8 +23,8 @@ Please be aware that these policies pertain to your software, content, and digit
 for use in connection with Google Chrome and distributed via the Chrome Web Store, including
 applications, extensions, and themes, as well as any ads bundled or made available through the
 product. These policies have the same meaning as in the [Google Chrome Web Store Developer
-Agreement][7]. There are additional quality guidelines that apply to [Chrome Apps][8] and [Chrome
-Extensions][9]. Be sure to check back from time to time, as these policies may change.
+Agreement][developer-agreement]. There are additional quality guidelines that apply to [Chrome Apps][section-chrome-apps] and [Chrome
+Extensions][section-extensions]. Be sure to check back from time to time, as these policies may change.
 
 ## Content Policies {: #content_policies }
```

Comment:
```suggestion
## Content policies {: #content_policies }
```

Reply:
Headings need to be sentence case. I'll correct what I can. Will you please check these docs headings that aren't visible in the PR?

Reply:
WIll do! 👍